README.md

```diff
@@ -102,8 +103,9 @@ distributions.
 
 ```bash
 opx-build/scripts/opx_run opx_rel_pkgasm.py --dist stable \
-  -b opx-onie-installer/release_bp/OPX_dell_base.xml
+  -b opx-onie-installer/release_bp/OPX_dell_base_<os_release>.xml
```

Comment:
Might be better to use an explicit example, and note what *can* be changed:

```bash
opx-build/scripts/opx_run opx_rel_pkgasm.py --dist stable \
  -b opx-onie-installer/release_bp/OPX_dell_base_stretch.xml
```

**Note**: Change `stretch` to `jessie` to build a Debian Jessie-based installer.